app/assets/stylesheets/functions/_shade.scss

```diff
@@ -0,0 +1,24 @@
+@charset "UTF-8";
+
+/// Mixes a color with black.
+///
+/// @param {Color} $color
+///
+/// @param {Number (Percentage)} $percent
+///   The amount of black to be mixed in.
+///
+/// @example scss - Usage
+///   .element {
+///     background-color: shade(#ffbb52, 60%);
+///   }
+///
+/// @example css - CSS Output
+///   .element {
+///     background-color: #664a20;
+///   }
+///
```

Comment:
I added a full example declaration here because I imagined people using it in this way. For things like the `contains` function, usually that would only be used within another function or mixin; not in your normal app styles.